docs/src/denudation/chemical.md

```diff
@@ -75,16 +85,28 @@ However, to solve this equation we still need to know $c(z)$.
 
 If assuming the initial percolating water has $c(0) = 0$, then we could get the following equation (as $c$ is related to depth):
 
-$$c(z) = c_{eq} * (1 - e^{(-z/\lambda)})$$
+$$c(z) = c_{eq}\ (1 - e^{(\frac{-z}{\lambda})})$$
```

Comment:
Superfluous parentheses. In superscript like this I prefer a/b notation for fractions